doc/source/data/working-with-ndarrays.rst

```diff
@@ -0,0 +1,140 @@
+.. _working_with_ndarrays:
```

Comment:
this needs to be added in the table of contents somewhere?

Reply:
It's in `user-guider.rst`.